java/conf/rhn_java.conf

```diff
@@ -226,7 +226,7 @@ java.mgr_sync_max_connections = 4
 java.max_changelog_entries = 20
 
 # Determins which product tree variation to use.
-java.product_tree_tag = SUMA5.0
+java.product_tree_tag = SUMA5.1
```

Comment:
shouldn't  we  adjust  also the lines later about the DB?
(Sorry,I can't comment on the right  lines)

Currently I see:
~~~
# minimal required DB schema version
java.min_schema_version = 5.0.8
# minimal required DB reporting schema version
java.min_report_schema_version = 5.0.5
~~~

Reply:
This will be done via the tito.props changes when you tag the package the next time.

Reply:
Still, it make sense to update the values here too from time to time I guess :)

Reply:
> This will be done via the tito.props changes when you tag the package the next time.

@mcalmer from when tito is changing  a value  in the rhn_java.conf?

Reply:
When you call "tito tag" it read tito.props and put these values into that config file. 
This is the way you can specify "future" schema versions without affecting acceptance testsuite.
If we define "minimal schema 5.1.3", but the still have schema 5.1.2, the java code will reject to connect to the DB.
But you will get 5.1.3 only when you tag.

I do not see a reason to manually change the version now. This is just a minimal version and as long as we are in development mode, we can live with it.